src/ManagedServices/ManagedServices/Az.ManagedServices.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.0.1'
+ModuleVersion = '1.0.0'
```

Comment:
Please revert this version change and update the changelog instead.  We will update the version in this case.